src/erlcloud_mon.erl

```diff
@@ -447,7 +447,29 @@ configure(AccessKeyID, SecretAccessKey, Host) ->
     put(aws_config, new(AccessKeyID, SecretAccessKey, Host)),
     ok.
 
-default_config() -> erlcloud_aws:default_config().
+default_config() ->
+    case get(aws_config) of
+        undefined ->
+            DefaultConfig = erlcloud_aws:default_config(),
```

Comment:
It seems like you can just use erlcloud_aws:service_config/3 here and delete most of this code?